gpiozero/boards.py

```diff
@@ -1532,6 +1532,37 @@ def __init__(self, pwm=False, pin_factory=None):
         )
 
 
+class TrafficpHat(TrafficLights):
+    """
+    Extends :class:`TrafficLights` for the `Pi Supply Traffic pHAT`_: Mini traffic
+    light LEDs.
+
+    The Traffic pHAT pins are fixed and therefore there's no need to specify
+    them when constructing this class. The following example then turns on all the LEDs::
+
+        from gpiozero import TrafficpHat
+        phat = TrafficpHat()
+        phat.red.on()
+        phat.blink()
+
+    :param bool pwm:
+        If :data:`True`, construct :class:`PWMLED` instances to represent each
+        LED. If :data:`False` (the default), construct regular :class:`LED`
+        instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _PiSupply Traffic pHAT: http://pisupp.ly/trafficphat
```

Comment:
@ryanteck This link seems to just redirect to https://uk.pi-supply.com/ ? :confused:

Reply:
It's currently unreleased so I had to use a placeholder which will re-direct once it is if that's ok?